content/agent/troubleshooting.md

```diff
@@ -85,27 +85,27 @@ In the commands below, replace `<CASE_ID>` with your Datadog support case ID, if
 
 ## Status of an Agent check
 
-Agent checks can't be directly called from python and instead need to be called by the Agent.
+Agent checks shouldn't be directly called from Python and instead must be called by the Agent.
```

Comment:
The use of "shouldn't" here reduces clarity. Is it:
* You can do this but you shouldn't
or
* You can't do this
?